### PR TITLE
release: add 1.5.5 release note

### PR DESCRIPTION
**Pull request by innobead**
#### Which issue(s) this PR fixes:

Issue longhorn/longhorn#8356

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.5.5.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+>  [!IMPORTANT]  
+> **Ensure that your cluster is running Kubernetes v1.21 or later before installing Longhorn v1.5.5.**
+
+You can install Longhorn using a variety of tools, including Rancher, Kubectl, and Helm. For more information about installation methods and requirements, see [Quick Installation](https://longhorn.io/docs/1.5.5/deploy/install/) in the Longhorn documentation.
+
+## Upgrade
+
+>  [!IMPORTANT]  
+> **Ensure that your cluster is running Kubernetes v1.21 or later before upgrading from Longhorn v1.5.x (< v1.5.5) or v1.4.x to v1.5.5.**
```
Comment (jillian-maroket) [inline CHANGELOG/CHANGELOG-1.5.5.md:20]:
```suggestion
> **Ensure that your cluster is running Kubernetes v1.21 or later before upgrading from Longhorn v1.4.x or v1.5.x to v1.5.5.**
```

Comment (jillian-maroket) [reply CHANGELOG/CHANGELOG-1.5.5.md:20]: When enumerating versions, we usually go from earliest to latest. Also, (< v1.5.5) is not necessary.

File: CHANGELOG/CHANGELOG-1.5.5.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+For information about important changes, including feature incompatibility, deprecation, and removal, see [Important Notes](https://longhorn.io/docs/1.5.5/deploy/important-notes/) in the Longhorn documentation.
+
+## Post-Release Known Issues
+
+For information about issues identified after this release, see [Release-Known-Issues](https://github.com/longhorn/longhorn/wiki/Release-Known-Issues).
+
+## Resolved Issues
+
+### Improvement
```
Comment (jillian-maroket) [inline CHANGELOG/CHANGELOG-1.5.5.md:34]:
```suggestion
### Improvements
```

File: CHANGELOG/CHANGELOG-1.5.5.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+- [BACKPORT][v1.5.5][IMPROVEMENT] Cannot read/write to block volume when the container is run as non-root [8123](https://github.com/longhorn/longhorn/issues/8123) - @PhanLe1010 @chriscchien
+- [BACKPORT][v1.5.5][IMPROVEMENT] Do not terminate nfs-ganesha in share-manager pod after failing to access recovery backend [8347](https://github.com/longhorn/longhorn/issues/8347) - @derekbit @chriscchien
+- [BACKPORT][v1.5.5][IMPROVEMENT] Expose virtual size of qcow2 backing images [8321](https://github.com/longhorn/longhorn/issues/8321) - @shuo-wu @chriscchien
+-  [BACKPORT][v1.5.5][IMPROVEMENT] Improve logging in CSI plugin when mount fails.  [8286](https://github.com/longhorn/longhorn/issues/8286) - @james-munson @chriscchien
+- [BACKPORT][v1.5.5][IMPROVEMENT] Upgrade support bundle kit version to v0.0.36 [8161](https://github.com/longhorn/longhorn/issues/8161) - @c3y1huang @roger-ryao
+- [BACKPORT][v1.5.5][IMPROVEMENT] Improve environment_check script for NFS protocol bug and the host system self diagnosis [7972](https://github.com/longhorn/longhorn/issues/7972) - @james-munson @roger-ryao
+
+### Bug
```
Comment (jillian-maroket) [inline CHANGELOG/CHANGELOG-1.5.5.md:42]:
```suggestion
### Bug Fixes
```

File: CHANGELOG/CHANGELOG-1.5.5.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+- [BACKPORT][v1.5.5][BUG] potential risk to unmap a negative number [8236](https://github.com/longhorn/longhorn/issues/8236) - @Vicente-Cheng @roger-ryao
+- [BACKPORT][v1.5.5][BUG] Use config map to update `default-replica-count` won't apply to `default-replica-count.definition.default` if the value equal to current `default-replica-count.value` [8135](https://github.com/longhorn/longhorn/issues/8135) - @james-munson @chriscchien
+- [BACKPORT][v1.5.5][BUG] LH manager reboots due to the webhook is not ready [8036](https://github.com/longhorn/longhorn/issues/8036) - @ChanYiLin @chriscchien
+- [BACKPORT][v1.5.5][BUG] Can't use longhorn with Generic ephemeral volumes [8201](https://github.com/longhorn/longhorn/issues/8201) - @ejweber @roger-ryao
+- [BACKPORT][v1.5.5][BUG] Volume cannot attach because of the leftover non-empty volume.status.PendingNodeID after upgrading Longhorn [7996](https://github.com/longhorn/longhorn/issues/7996) - @james-munson @roger-ryao
+- [BACKPORT][v1.5.5][BUG] no Pending workload pods for volume xxx to be mounted [8082](https://github.com/longhorn/longhorn/issues/8082) - @c3y1huang @roger-ryao
+
+### Misc
```
Comment (jillian-maroket) [inline CHANGELOG/CHANGELOG-1.5.5.md:59]:
```suggestion
### Miscellaneous
```

**Review by jillian-maroket — COMMENTED** [review]
Just a few nits

File: CHANGELOG/CHANGELOG-1.5.5.md (hunk 1 above)
Comment (derekbit) [reply CHANGELOG/CHANGELOG-1.5.5.md:20]: @jillian-maroket  Explicit add `(< v1.5.5)` might be better for avoiding confusion. WDYT?

Comment (jillian-maroket) [reply CHANGELOG/CHANGELOG-1.5.5.md:20]: The word "upgrading" already implies moving from an earlier version to a later version. In this case, 1.5.x implies all versions released before 1.5.5. I'm not sure what kind of confusion is going to happen but feel free to add it back if you want.

Comment (derekbit) [reply CHANGELOG/CHANGELOG-1.5.5.md:20]: > The word "upgrading" already implies moving from an ...

I see. Thanks for the explanation.
IMHO, making it explication is not bad, and it is clear and helpful for some users that might "upgrade" from 1.5.6 to 1.5.5...

**Review by derekbit — APPROVED** [review]

**Comment by innobead** [comment]
Thanks @derekbit @jillian-maroket

**Pull request merged**